### ISSUE TITLE
Quest: Blueleaf Tubers

### ISSUE THREAD

**Issue by transroute** [dalaranwow/dalaran-wow#2911]
When you accept the quest, it is asked of you to pick up certain things. 
But when you try to open the book, to get the User Manual, it opens. but there is nothing
in there - so you can not pick it up...

**Comment by Spruitje** [comment]
Thats weird, i did this quest not long ago and though it had some issues (https://github.com/dalaranwow/dalaran-wow/issues/318) i was able to complete this quest
I had the users manual, which is just a book you pickup with text on how to use the controlsticks on the gophers, i even read this ingame book since my experience on that quest felt buggy
I did this on alliance, though i don't expect faction would matter much
Basically you get 3 items in your bags: The owners manual(a book), Crate with holes(limited charges, summons gophers) and the controlstick
Iirc the picking up the book does count as a objective for the quest, you will see in your questlog if this was succesfull, however the book is not needed to perform any actions during the quest, and should be returned/disappear upon turning in the quest

**Comment by Golganeth** [comment]
Fixed in-game. This is a rare and general issue with game objects. The quest itself works.

**Comment by transroute** [comment]
Yes, just a sec ago I picked up everything.
